contrib/cirrus/packer/fedora_setup.sh

```diff
@@ -31,6 +31,7 @@ ooe.sh sudo dnf install -y \
     bridge-utils \
     btrfs-progs-devel \
     bzip2 \
+    conmon \
```

Comment:
So, older versions of podman didn't require the package, but conmon was still installed as part of the main podman package itself at `/usr/libexec/podman/conmon`. Just making sure we're on the same page.

Reply:
oh yes, and thanks for the reminder, there's a TODO I should also strip out...

Reply:
...done.